src/components/ContentEditable.js

```diff
@@ -21,65 +22,75 @@ class ContentEditable extends React.Component{
   }
 
   componentDidMount(){
-    this.setFocus()
+    this.handleFocusAndBlur()
   }
 
-  componentDidUpdate(){
-    this.setFocus()
+  componentDidUpdate(oldProps, oldState){
+    this.handleFocusAndBlur(oldProps, oldState)
   }
     
-  setFocus = () => {
+  handleFocusAndBlur = (oldProps, oldState) => {
     if(this.props.currentElem.elemId === this.props.id){
       if(this.elem)
         this.elem.focus()
     }
+    else if(oldProps && this.props.currentElem.elemId === oldProps.currentElem.elemId){
+      if(this.elem)
+        this.elem.blur()
+    }
   }
 
-  emitChange = (e, context) => {
+  emitChange = (e) => {
     if (!this.props.componentType && e.target.innerHTML) {
-      context.emitUpdate(null, { content: e.target.innerHTML }, 'updateTitle')
+      this.context.emitUpdate(null, { content: e.target.innerHTML }, 'updateTitle')
     }                   // Block to make changes to title of the page
   }
 
-  handleMouseUp = (e) => {
-    if(!this.props.componentType)
-      this.props.setCurrentElem(this.props.id)
-  }
-  
   handleFocus = (e) => {
     e.persist()
     if(!this.props.componentType)
       setCursorToEnd(e)
   }
 
   handleNewLine = (e) => {
-    if(e.keyCode === 13 && this.props.id === `page-title`) {
+    if(e.key === 'Enter' && this.props.id === `page-title`) {
```

Comment:
What is wrong with using keyCode here ?

Reply:
1) keyCode has been deprecated 
2) better readability.